config/kibana.yml

```diff
@@ -111,3 +111,28 @@
 
 # Specifies locale to be used for all localizable strings, dates and number formats.
 #i18n.locale: "en"
+
+# Set to false to disabled the APM plugin Kibana. Defaults to true.
+#xpack.apm.enabled: true
+
+# Set to false to hide the APM plugin Kibana from the menu. Defaults to true.
+#xpack.apm.ui.enabled: true
```

Comment:
We should double check with someone from Kibana whether it's okay to add `xpack` configs to the default config file (I hope it is).

@tylersmalley ?

Reply:
Unfortunately, I don't feel we should include these as this file is also used with the OSS distribution.  What is the motivation for having them in the sample config?

Reply:
Additionally, Elasticsearch also does not include xpack specific configs either - rather, only a small set of extreamly common configuration options: https://github.com/elastic/elasticsearch/blob/master/distribution/src/config/elasticsearch.yml

Reply:
> What is the motivation for having them in the sample config?

Helping users to discover the options without having to go to the docs. But we'll undo this for now.